src/pages/manage_dac/ManageDac.js

```diff
@@ -1,420 +1,214 @@
-                    }, ['Edit']),
-                    h(TableIconButton, {
-                      key: 'delete-dac-icon',
-                      dataTip: disabled ? 'All datasets assigned to this DAC must be reassigned before this can be deleted' : 'Delete DAC',
-                      isRendered: userRole === ADMIN,
-                      disabled: disabled,
-                      onClick: () => this.deleteDac(dac),
-                      icon: Delete,
-                      style: Object.assign({}, Styles.TABLE.TABLE_ICON_BUTTON),
-                      hoverStyle: Object.assign({}, Styles.TABLE.TABLE_BUTTON_ICON_HOVER)
-                    })
-                  ])
-                ]),
-                hr({className: 'table-body-separator'})
-              ])
-              );
-            }),
-          PaginatorBar({
-            total: this.state.dacs.filter(this.searchTable(searchDacText)).length,
-            limit: this.state.limit,
-            pageCount: this.pageCount,
-            currentPage: this.state.currentPage,
-            onPageChange: this.handlePageChange,
-            changeHandler: this.handleSizeChange
-          }),
-          DacMembersModal({
-            isRendered: this.state.showMembersModal,
-            showModal: this.state.showMembersModal,
-            onOKRequest: this.closeViewMembersModal,
-            onCloseRequest: this.closeViewMembersModal,
-            dac: this.state.selectedDac
-          }),
-          DacDatasetsModal({
-            isRendered: this.state.showDatasetsModal,
-            showModal: this.state.showDatasetsModal,
-            onOKRequest: this.closeViewDatasetsModal,
-            onCloseRequest: this.closeViewDatasetsModal,
-            dac: this.state.selectedDac,
-            datasets: this.state.selectedDatasets
-          }),
-          AddDacModal({
-            isRendered: this.state.showDacModal,
-            showModal: this.state.showDacModal,
-            isEditMode: this.state.isEditMode,
-            onOKRequest: this.okAddDacModal,
-            onCloseRequest: this.closeAddDacModal,
-            dac: this.state.selectedDac,
-            userRole: userRole
-          }),
-          h(ConfirmationModal, {
-            showConfirmation: this.state.showConfirmationModal,
-            closeConfirmation: this.closeConfirmation,
-            title: 'Delete DAC?',
-            message: 'Are you sure you want to delete this Data Access Committee?',
-            header: this.state.selectedDac.name,
-            onConfirm: () => this.handleDeleteDac(),
-          }),
-          h(ReactTooltip, {
-            place: 'left',
-            effect: 'solid',
-            multiline: true,
-            className: 'tooltip-wrapper'
-          })
+        ]
+      ),
+      div({className: 'right-header-section'}, [
+        a({
+          id: 'btn_addDAC',
+          className: 'col-md-12 btn-primary btn-add common-background',
+          style: {
+            marginTop: '30%'
```

Comment:
nit: I'm seeing the button getting distorted at a screen width of ~990px. adding `display: 'flex'` here should fix it